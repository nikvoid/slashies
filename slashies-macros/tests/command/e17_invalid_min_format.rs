use slashies_macros::Command;

/// An invalid command
#[derive(Command)]
#[name = "BadCommand"]
struct BadCommand {
    /// Field with badly formatted min attribute
    #[min]
    bad_field: i64,

    /// Ok field
    #[min = 0]
    good_field: i64,
}

fn main() {}
