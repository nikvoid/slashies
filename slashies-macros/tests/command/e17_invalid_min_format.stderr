error: Invalid "min" attribute. Attribute must be of the form #[min = value]
 --> tests/command/e17_invalid_min_format.rs:8:5
  |
8 |     #[min]
  |     ^^^^^^
